web/web.bzl

```diff
@@ -171,18 +169,33 @@ def browser(testonly=True, **kwargs):
   browser_alias(testonly=testonly, **kwargs)
 
 
-def web_test(browser, test, data=None, size=None, **kwargs):
+def web_test(browser,
+             test,
+             config=None,
+             data=None,
+             launcher=None,
+             size=None,
+             **kwargs):
   """Wrapper around web_test to correctly set defaults.
   
   Args:
     data
     size: default = "large"
     **kwargs: see web_test in //web/internal:web_test.bzl
   """
+  config = config or str(Label("//web:default_config"))
+  launcher = launcher or str(Label("//go/launcher:main"))
   data = lists.clone(data)
```

Comment:
That is similar to what I was doing before, but I found that it doesn't work from some rule types (specifically I remember that when test referred to a py_test it didn't work because the py runfiles has an empty file provider for __init__.py files that gets missed by this).